security/src/main/java/org/wildfly/security/soteria/original/AuthenticationController.java

```diff
@@ -138,7 +145,7 @@ private void storeRequestURL(HttpServletRequest request, HttpServletResponse res
         }
     }
 
-    private String getFullURL(HttpServletRequest request) {
+    private  String getFullURL(HttpServletRequest request) {
```

Comment:
This shouldn't block anything, but looks like just a white space change.

Reply:
thank you, I have resisted formatting updates this time other than as needed to get past checkstyle as it make it hard to compare this forked code with the original - Soteria has some bad formatting like this.